pkg/sql/codegen/pai/template_tf.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pai
+
+type submitFiller struct {
+	DataSource string
+	EntryFile  string
+	ModelName  string
+}
+
+type saveModelFiller struct {
+	DataSource   string
+	ModelName    string
+	Estimator    string
+	Save         string
+	IsKerasModel bool
+}
+
+type predictFiller struct {
+	DataSource  string
+	ModelName   string
+	Select      string
+	ResultTable string
+}
+
+const tfSubmitTmplText = `
```

Comment:
If use `pyodps` but not `odpscmd` can we have a submitter program under `python/sqlflow_submitters` to make this template simpler?

Reply:
I'm afraid we can't. `pyodps` cannot submit xflow tasks to our cluster. The reason is still under investigation.

Reply:
I see.